questionnaires/chair_rises_instructions/chair_rises_instructions_armt.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "field_name": "chair_rises_instructions",
+        "form_name": "chair_rises",
+        "section_header": "",
+        "field_type": "descriptive",
+        "field_label": "<p><strong>Instruction Video</strong><br /><br /><iframe src=\"https://radar-armt-assets.s3.eu-west-2.amazonaws.com/nshd_chair_rise_vid.mp4\" width=\"280\" height=\"315\" allowfullscreen=\"\"></iframe><br /><br />For further instructions,<br>please <a href=\"https://nshd.mrc.ac.uk/wearables-and-remote-testing-written-instructions/\">visit this link.</a></p>",
+        "select_choices_or_calculations": "",
```

Comment:
Did they want this to be a separate questionnaire? Will this be scheduled just before the chair rises task? Or will these be in the clinical tasks page to view anytime?

Reply:
Yup this will be in the clinical tasks page to view anytime.